tools/release/notify-changelog-cut.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e
+
+# Ensure "gh" tool is installed.
+if ! command -v gh &> /dev/null; then
+    echo "The 'gh' command cannot be find. Please install it: https://cli.github.com"
+    exit
+fi
+
+# Config
+NOTIFICATION_LABEL="release/notified-changelog-cut"
+
+# List all open PRs.
+OPEN_PR_IDS=$(gh pr list --repo grafana/mimir --limit 1000 --state open --json number,title --jq '.[].number')
+
+for PR_ID in ${OPEN_PR_IDS}; do
+  # Get PR details
+  PR_TITLE=$(gh pr view --comments "${PR_ID}" --json title --jq '.title')
+  PR_LABELS=$(gh pr view --comments "${PR_ID}" --json labels --jq '.labels[].name')
+
+  # Log the PR we're currently processing.
+  echo "- ${PR_ID}: ${PR_TITLE}"
+
+  # Check if the notification has already been posted.
+  # We suppress the linter warning about looking for the extract substring instead of doing a regex
+  # match, but that's exactly what we want.
+  # shellcheck disable=SC2076
+  if [[ ${PR_LABELS} =~ "${NOTIFICATION_LABEL}" ]]; then
+    echo "  Notification already posted, nothing to do"
+    continue
+  fi
```

Comment:
This won't work for PRs spanning for several releases, but not sure if that's important to cover.

Reply:
True. Let's see if it will be a real issue in the future :)

Reply:
Done is better than perfect! 😉